docs/usage/misc.rst

```diff
@@ -0,0 +1,168 @@
+.. _usage-misc-label:
+
+====
+Misc
+====
+
+
+Plot fit results
+================
+
+Two steps are necessary when you want to use ``plothist`` to plot the result of a fit using ``RooFit`` or ``zfit``.
+
+Getting the PDF
+---------------
+
+First you need to convert your functions to scipy functions. This can be done using a small function.
+
+With RooFit
+~~~~~~~~~~~
+
+This should be called after you have fitted your model and you have a ``RooAbsPdf`` object.
+
+.. code-block:: python
+
+   import numpy as np
+   from scipy.interpolate import interp1d
+   import pickle
+
+   def save_pdf(var, pdf, path="pdf.pkl", npoints=10000, verbose=True):
+      """
+      Save a RooFit PDF as a scipy.interpolate.interp1d function.
+
+      Parameters
+      ----------
+      var : RooRealVar
+         The variable to evaluate the PDF at.
+      pdf : RooAbsPdf
+         The PDF to save.
+      path : str, optional
+         The path to save the PDF to. Should end with `.pkl`. Default is "pdf.pkl".
+      npoints : int, optional
+         The number of points to evaluate the PDF at. Default is 10000.
+      verbose : bool, optional
+         Whether to print progress. Default is True.
+
+      Returns
+      -------
+      pdf_func : scipy.interpolate.interp1d
+         The PDF as a function.
+
+      Notes
+      -----
+      The PDF is saved as a scipy.interpolate.interp1d function with pickle.
+      """
+      pdf_x = []
+
+      xlim = (var.getMin(), var.getMax())
+      # Get a sample of x values
+      x = np.linspace(*xlim, npoints)
+
+      for k, i in enumerate(x, 1):
+         var.setVal(i)
+         # Evaluate the PDF at the given x value
+         pdf_x.append(pdf.getVal(var))
+         if verbose:
+               print(f"\t{path.replace('.pkl','')} progress [{k*100/len(x):.1f}%]", end="\r")
+      if verbose: print()
+
+      # Interpolate the PDF
+      pdf_func = interp1d(x, pdf_x)
```

Comment:
Can we avoid fitting and save the real function?

Reply:
I don't think there are easier ways to do that. We want to save a scipy object to input in `plothist`, not a RooFit object.